src/main/java/Duke.java

```diff
@@ -1,10 +1,81 @@
+//Entirely refurbishing Duke to OOP based in the nxt version :)
+import java.util.Scanner;
 public class Duke {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws EmptyTodoException, IncorrectInputException {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        String border = "____________________________________________________________\n";
+        System.out.print(border + " Hello! I'm Duke\n" + " What can I do for you?\n" + border);
+        String command;
+        int size = 0;
+        String temp;
+        String tag = "[ ]";
+        String[] temp2 = new String[2];
```

Comment:
"temp2" is a vague name - consider following the format to convey the exact purpose